### ISSUE TITLE
disallow ulem package

### ISSUE THREAD

**Issue by kmccurley** [IACR/latex#246]
It seems that the `ulem` package causes havoc with our style, and I have removed it from the docker image. Unfortunately there are a fair number of other packages that depend on ulem, including [`changes.sty`](https://ctan.org/pkg/changes), [`coop-writing`](https://ctan.org/pkg/coop-writing), `fixmetodonotes`, `colorist`, etc. Moreover the features of the `ulem` package seem pretty useful. Apparently this problem with our style was to interfere with the behavior of `\em`, but the author of the `ulem` package offers the option to load with `\usepackage[normalem]{ulem}` so that it doesn't interfere with the `\em` command. One possible solution is for us to load this way, and another is to check to see if `ulem` is loaded at the end of the preamble, and then call `\normalem`.

**Comment by kmccurley** [comment]
I found a couple of papers that appear to rely upon commands from the `ulem` package, namely `\sout` for strikethrough. If this package interferes with `iacrcc.cls`, then we might consider checking after the preamble whether `ulem` was loaded and if so call `\normalem`. This is the [suggested method](https://mirrors.mit.edu/CTAN/macros/latex/contrib/ulem/ulem.pdf) for the `ulem` package.

**Comment by jwbos** [comment]
Indeed, better to support `ulem` and when it is used amke sure the style is not affected. 
Fixed in 8ff11da12ea95df2c84bdc7ceaf97871da08a4c2.
